modules/simple_forms_api/app/form_mappings/vba_21_0845.json.erb

```diff
@@ -0,0 +1,93 @@
+  "F[0].#subform[2].ZIPOrPostalCode_LastFourNumbers[0]": "<%= data.dig('organization_address', 'postal_code')&.[](0..4) %>",
+  "F[0].#subform[2].ZIPOrPostalCode_FirstFiveNumbers[0]": "<%= data.dig('organization_address', 'postal_code')&.[](5..8) %>",
+
+  "F[0].#subform[2].LIMITED_INFORMATION[0]": "<%= data['information_scope'] == 'limited' ? 1 : 0 %>",
+  "F[0].#subform[2].ANY_INFORMATION[0]": "<%= data['information_scope'] == 'any' ? 1 : 0 %>",
+
+  "F[0].#subform[2].Status_Of_Pending_Claim_Or_Appeal[0]": "<%= data['limited_information_items']&.include?('Status of pending claim or appeal') ? 1 : 0 %>",
+  "F[0].#subform[2].CURRENT_BENEFIT_AND_RATE[0]": "<%= data['limited_information_items']&.include?('Current benefit and rate') ? 1 : 0 %>",
+  "F[0].#subform[2].PAYMENT_HISTORY[0]": "<%= data['limited_information_items']&.include?('Payment history') ? 1 : 0 %>",
+  "F[0].#subform[2].AMOUNT_OF_MONEY_OWED_VA[0]": "<%= data['limited_information_items']&.include?('Amount of money owed to VA') ? 1 : 0 %>",
+  "F[0].#subform[2].REQUEST_A_BENEFIT_PAYMENT_LETTER[0]": "<%= data['limited_information_items']&.include?('Request a benefit payment letter') ? 1 : 0 %>",
+  "F[0].#subform[2].CHANGE_OF_ADDRESS_OR_DIRECT_DEPOSIT[0]": "<%= data['limited_information_items']&.include?('Change of address or direct deposit') ? 1 : 0 %>",
+  "F[0].#subform[2].OTHER_Specify_Below[0]": "<%= data['limited_information_items']&.include?('Other (Specify below)') ? 1 : 0 %>",
+  "F[0].#subform[2].International_Telephone_Number_If_Applicable[5]": "<%= data['limited_information_other'] %>",
+
+  "F[0].#subform[2].One_time_only[0]": "<%= data['release_duration'] == 'onceOnly' ? 1 : 0 %>",
+  "F[0].#subform[2].ONGOING_UNTIL_WRITTEN_NOTICE_IS_GIVEN_TO_VA_TO_TERMINATE[0]": "<%= data['release_duration'] == 'untilNotice' ? 1 : 0 %>",
+  "F[0].#subform[2].FROM_THE_DATE_OF_SIGNING_BELOW_UNTIL[0]": "<%= data['release_duration'] == 'untilDate' ? 1 : 0 %>",
+  "F[0].#subform[2].End_Date_Year[0]": "<%= data['release_end_date']&.[](0..3) %>",
+  "F[0].#subform[2].End_Date_Day[0]": "<%= data['release_end_date']&.[](8..9) %>",
+  "F[0].#subform[2].END_DATE_Month[0]": "<%= data['release_end_date']&.[](5..6) %>",
+
+  "F[0].#subform[2].THE_CITY_AND_STATE_YOUR_MOTHER_WAS_BORN_IN[0]": "<%= data['security_question'] == 'motherBirthplace' ? 1 : 0 %>",
+  "F[0].#subform[2].THE_NAME_OF_THE_HIGH_SCHOOL_YOU_ATTENDED[0]": "<%= data['security_question'] == 'highSchoolName' ? 1 : 0 %>",
+  "F[0].#subform[2].YOUR_FIRST_PETS_NAME[0]": "<%= data['security_question'] == 'firstPetName' ? 1 : 0 %>",
+  "F[0].#subform[2].YOUR_FAVORITE_TEACHERS_NAME[0]": "<%= data['security_question'] == 'favoriteTeacherName' ? 1 : 0 %>",
+  "F[0].#subform[2].YOUR_FATHERS_MIDDLE_NAME[0]": "<%= data['security_question'] == 'fatherMiddleName' ? 1 : 0 %>",
+  "F[0].#subform[2].International_Telephone_Number_If_Applicable[0]": "<%= data['security_question'] == 'motherBirthplace' ? data['security_answer'] : '' %>",
```

Comment:
is this section about international telephone correct? it looks like this is referring to security question

Reply:
Yeah this part was confusing for me since the fields are very mislabeled. But I've checked these and they match up correctly.